indicators/indicators.py

```diff
@@ -122,11 +129,12 @@ def compute_indicator(da, idx, coord_labels, kwargs={}):
     Returns:
         A new data array with dimensions year, latitude, longitude, in that order containing the summarized information
     """
+    # dask array must be computed here in order to change nodata values
     new_da = (
         globals()[idx](da, **kwargs)
         # .transpose("time", "lat", "lon")
         # .reset_coords(["longitude", "latitude", "height"], drop=True)
```

Comment:
Is there a reason we want to keep this commented out code? If not, I vote that we remove it before merging.